kroxylicious-filters/kroxylicious-encryption/src/main/java/io/kroxylicious/filter/encryption/KeyManager.java

```diff
@@ -21,18 +24,18 @@
 public interface KeyManager<K> {
 
     /**
-     * Asynchronously encrypt the given {@code recordRequests} using the current DEK for the given KEK, calling the given receiver for each encrypted record
+     * Asynchronously encrypt the given {@code records} using the current DEK for the given KEK returning a transformed record batch
```

Comment:
talking about "record batch" here is incorrect, memory records contains multiple batches